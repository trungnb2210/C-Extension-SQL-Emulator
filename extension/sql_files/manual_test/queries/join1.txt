SELECT name, balance, saving
FROM people JOIN bankAccounts ON name == holder
WHERE balance + saving > 200
