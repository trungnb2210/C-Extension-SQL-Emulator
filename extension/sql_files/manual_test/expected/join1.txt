QUERY RESULT:
  name    balance   saving 
----------------------------
| Bob   | 500     | 999    | 
| Angus | 200     | 80     | 
| Angus | 200     | 80     | 
| Angus | 200     | 80     | 
----------------------------
